src/components/PartnersContent/PartnersTile/PartnersTile.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from "react";
+import { Button, ButtonSize } from "@govtechsg/tradetrust-ui-components";
+import { PartnersTileProps } from "../types";
+
+export const PartnersTile: FunctionComponent<PartnersTileProps> = ({ data }) => {
+  const { companyName, description, logo, websiteLink } = data;
+  return (
+    <div className="max-w-[380px] min-w-[380px] bg-white rounded-xl shadow-md p-5 mb-10 mr-8">
+      <p className="text-2xl">{companyName}</p>
+      <div className="max-h-[148px] min-h-[148px] flex justify-center">
+        <img src={logo} className="mx-auto" data-testid={`partner-logo-${companyName}`} />
```

Comment:
👀

Reply:
?

Reply:
i dont want the logo to be 1 big or 1 small so i want to fix the height for the logo

Reply:
u know la, the imda user wont photoshop the logo to all exact hight one

Reply:
add `flex-col`

Reply:
ok updated